module/src/components/config/config.context.tsx

```diff
@@ -97,7 +97,7 @@ const systemDefaults: Required<IArmstrongConfig> = {
   validationErrorIcon: <IoIosWarning size={24} />,
   spinnerIcon: <ImSpinner2 />,
   dialogCloseButtonIcon: <RiCloseLine size={24} />,
-  globalPortalTo: document.body,
+  globalPortalTo: document && document.body || undefined,
```

Comment:
I think you might want to do one of these boys `typeof document !== 'undefined'` which is the ol classic Gatsby/Next way of checking

Reply:
Yeah this probably needs to be:
```
globalPortalTo: typeof document !== 'undefined' ? document : undefined
```

Reply:
Because the IArmstrongConfig type is wrapped in a Required generic it can't be set to undefined, I thought of 2 different ways to go about this

1) Allow undefined in the main config
2) Omit globalPortalTo directly from the required generic
```
const systemDefaults: Required<Omit<IArmstrongConfig, 'globalPortalTo'>> & Pick<IArmstrongConfig, 'globalPortalTo'> = {};
```
3) Tell typescript to ignore the error
`globalPortalTo: typeof document !== 'undefined' && document.body || undefined!,`